scripts/apps/vocabularies/views/vocabulary-config-modal.html

```diff
@@ -60,7 +60,7 @@ <h3 class="modal__heading" ng-show="!vocabulary._id"><span translate>Create</spa
                 </div>
 
                 <div class="sd-line-input">
-                    <sd-tag-input ng-model="vocabulary.tags" data-label="Tags" data-items="tags" data-field="text" data-freetext="true" class="ng-pristine ng-untouched ng-valid ng-isolate-scope">
+                    <sd-tag-input ng-model="vocabulary.tags" data-label="{{Tags | translate}}" data-items="tags" data-field="text" data-freetext="true" class="ng-pristine ng-untouched ng-valid ng-isolate-scope" translate>
                         <tags-input ng-model="model" add-from-autocomplete-only="{{items ? !freetext : false}}" show-button="{{!!items}}" display-property="{{field}}" key-property="{{keyProperty}}" use-strings="{{!field}}" ng-required="required" on-tag-added="onChange($tag)" on-tag-removed="onChange($tag)" replace-spaces-with-dashes="{{replaceSpacesWithDashes}}" min-length="{{minLength}}" max-length="{{maxLength}}" placeholder="{{placeholder}}" class="ng-pristine ng-untouched ng-valid"></tags-input>
```

Comment:
translate directive can only be used for elements containing plain/text content. I don't think you need it on `<sd-tag-input>`

Reply:
I think the translation is applied to label only.

Reply:
You are already applying a filter for the label which is doing the translation. You don't need `translate` directive at the end. If you put it there grunt task will extract the whole `angularjs-html` as translation key to send to Transifex.

Reply:
done